Formula/asciidoc.rb

```diff
@@ -34,7 +31,7 @@ def install
     inreplace %w[a2x.py asciidoc.py filters/code/code-filter.py
                  filters/graphviz/graphviz2png.py filters/latex/latex2img.py
                  filters/music/music2png.py filters/unwraplatex.py],
-      "#!/usr/bin/env python2", "#!/usr/bin/python"
+      "#!/usr/bin/env python3", "#!#{Formula["python@3.8"].opt_bin}/python3"
```

Comment:
Try something like this

      bin.find { |f| rewrite_shebang detected_python_shebang, f }

Reply:
I get this error: `NameError: undefined local variable or method `detected_python_shebang'`

This would also not affect any of the `filters/` files which are kept under `etc/asciidoc/filters/` I think, so would require a similar thing to rewrite them?

Reply:
You need to `include Language::Python::Shebang` at the start of the file. And you should iterate on the array that's already there instead of `bin.find` it was just an example.

Reply:
Ah, thank you. Replaced.